ReleaseHistory.md

```diff
@@ -20,9 +20,7 @@
 
 ## v4.5.8 UNRELEASED
 - DEP: Update SARIF SDK submodule from [7e8def7 to dd3741f(https://github.com/microsoft/sarif-sdk/compare/7e8def7..dd3741f). [Full SARIF SDK release history](https://github.com/microsoft/sarif-sdk/blob/dd3741f/ReleaseHistory.md).
-- DEP: Upgrade `Microsoft.Security.Utilities` from 6.2.1 to 6.5.0. [#788](https://github.com/microsoft/sarif-pattern-matcher/pull/788)
```

Comment:
>macOS [](http://example.com/codeflow?start=21&length=5)

This is an unbelievably fantastic change. 

Did you delete some 4.5.8 notes, though?

Reply:
oh you are looking at 108 not latest 109, 109 is the one after i merged from main.

Reply:
there are a few additional merge from main action happened.